### PR TITLE
chore: fix unparsable lines in monitor command

### LINKED ISSUE #894: Some Admission Log Lines are not being Parse as JSON and likewise not Unmarshalled into LogEntry structs
### Describe what should be investigated or refactored

Pino should dump logs as JSON objects. The Admission Controller creates [similar logs when it processes admission objects](https://github.com/defenseunicorns/pepr/blob/c88df4fb834127e0b04019197d2034625b93238d/src/lib/controller/index.ts#L238), for some reason it seems like some of the logs are not being formatted in a way that can be parsed or unmarshalled. 

We need to investigate why.

### Links to any relevant code
Pepr Monitor - [Parse Line](https://github.com/defenseunicorns/pepr/blob/8fef88d328991237eb1bb47b03f82128b03af81c/src/cli/monitor.ts#L55)
uds monitor command - [Unmarshall line](https://github.com/defenseunicorns/uds-cli/blob/cd498f67313fd770890966d20e8e62b92482977d/src/pkg/engine/pepr/stream.go#L191)

### Additional context
Add any other context or screenshots about the technical debt here.

https://defense-unicorns.slack.com/archives/C06QJAUHWFN/p1718314201953779?thread_ts=1718269598.808709&cid=C06QJAUHWFN

```
IGNORED - Unable to parse line: {"level":30,"time":1718308326469,"pid":15,"hostname":"pepr-static-test-b4877b779-z2rcz","uid":"b7bce167-d82f-4b28-972e-d707528cb66b","namespace":"pepr-demo","name":"/example-evil-cm","res":{"uid":"b7bce167-d82f-4b28-972e-d707528cb66b","warnings":[],"allowed":true},"msg":"Check response"}. 
```

### PR DESCRIPTION
**Pull request by cmwylie19**
## Description

After debugging, adding in checks to ensure fields are defined and a slight sleep which helps to get complete JSON. I have not experience the issue since implementing the fix.

The issue #894  details the findings from debugging in a [comment](https://github.com/defenseunicorns/pepr/issues/894#issuecomment-2180688949)

## Related Issue

Fixes #894 

Relates to #

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Other (security config, docs update, etc)

## Checklist before merging

- [x] Test, docs, adr added or updated as needed
- [x] [Contributor Guide Steps](https://docs.pepr.dev/main/contribute/#submitting-a-pull-request) followed

### CODE REVIEW

File: src/cli/monitor.ts [hunk 1]
```diff
@@ -42,51 +42,54 @@ export default function (program: RootCmd) {
...
-              } else {
-                const failures = Array.isArray(payload.res) ? payload.res : [payload.res];
-
-                const filteredFailures = failures
-                  .filter((r: ResponseItem) => !r.allowed)
-                  .map((r: ResponseItem) => r.status.message);
-                if (filteredFailures.length > 0) {
-                  console.log(`\n❌  VALIDATE   ${name} (${uid})`);
-                  console.log(`\u001b[1;31m${filteredFailures}\u001b[0m`);
+        sleep(2, () => {
```
Comment (schaeferka) [inline src/cli/monitor.ts]: Should this be something like `await sleep(2);`?

Comment (cmwylie19) [reply src/cli/monitor.ts]: yes that is a good idea

**Review by schaeferka — DISMISSED** [review]
I like the update to the sleep line.

**Comment by cmwylie19** [comment]
This may not be the most optimal solution, we likely will need to revisit this in the future. For now we have a journey test and it mitigates the immediate issue. We may need to re-open and look at iterators possibly.

**Review by btlghrants — APPROVED** [review]
LGTM.

**Pull request merged**